src/universal/components/TeamInvitationEmailCreateAccount.tsx

```diff
@@ -26,17 +27,26 @@ const TeamName = styled('span')({
 
 const TeamInvitationEmailCreateAccount = (props: Props) => {
   const {verifiedInvitation} = props
-  const {teamName, teamInvitation} = verifiedInvitation
+  const {meetingType, teamName, teamInvitation} = verifiedInvitation
   if (!teamInvitation) return null
   const {email} = teamInvitation
   return (
     <StyledDialog>
       <Helmet title={`Sign up | Team Invitation`} />
-      <DialogTitle>Welcome!</DialogTitle>
+      <DialogTitle>
+        {meetingType ? `Join ${meetingTypeToLabel[meetingType]} Meeting in Progress` : 'Join Team'}
+      </DialogTitle>
       <DialogContent>
-        <InvitationDialogCopy>
-          Choose a password for immediate access to your team: <TeamName>{teamName}</TeamName>
-        </InvitationDialogCopy>
+        {meetingType ? (
+          <InvitationDialogCopy>
+            You’re just 1 step away from participating with your team:{' '}
```

Comment:
This says 1 step, but it doesn't say what that 1 step is. I prefer "Choose a password for immediate access" here
You can also reduce the size of the conditional:

```
<InvitationDialogCopy>
{meetingType ? meetingCopy : teamCopy}
<TeamName>{teamName}</TeamName>
 </InvitationDialogCopy>
```